src/style.css

```diff
@@ -0,0 +1,141 @@
+/* #region Normalize */

```

Comment:
use separate files for each BEM block instead of regions

Reply:
changed